core/templates/services/svg-sanitizer.service.ts

```diff
@@ -180,6 +180,9 @@ export class SvgSanitizerService {
     svg.querySelectorAll('*').forEach((node) => {
       let nodeTagName: string = node.tagName.toLowerCase();
       if (tagsToBeRemoved.indexOf(nodeTagName) !== -1) {
+        // The parent node will always be the svg tag itself.
```

Comment:
Not sure this is correct, can't there be nested tags (and for the children in those tags, the parent won't be the svg tag)?

Also, if you know that a parentNode will always exist, then why still keep the '?'

Reply:
I have updated the comment to mention "outermost" instead of "parent". 

Regarding the '?', we know that the case with an invalid outermost tag won't arise(as per the discussion above) but it is there to bypass the typescript strict check that object might be null.